src/components/ProductCard/ProductCard.stories.tsx

```diff
@@ -13,13 +13,12 @@ const Template: StoryFn<typeof ProductCardComponent> = (args) => (
 export const ProductCard = Template.bind({});
 ProductCard.args = {
   product: {
-    brand: "lancome",
-    title: "Advanced serum",
-    nrReviews: 9,
-    src: "https://www.sephora.ro/on/demandware.static/-/Sites-masterCatalog_Sephora/default/dw57c2ffcb/images/hi-res/all/SKU/308937_swatch.jpg",
-    alt: "serum",
-    href: "https://www.sephora.ro/p/advanced-genifique-serum---serum-activator-de-tinerete-P10007363.html",
-    height: 250,
-    width: 250,
+    brand_name: "lancome",
+    name: "Advanced serum",
+    images: [],
+    description: "",
+    ingredients: "",
```

Comment:
It's weird to have the description and ingredients empty. I see that the `<ProductCard>` component doesn't use them.

Then, I suggest we change the typings to reflect that `description` and `ingredients` are not used by the ProductCard component, using `Omit`.

```
type Props = {
  product: Omit<Product, 'description' | 'ingredients'>;
};
```